.github/CODEOWNERS

```
@@ -2,38 +2,38 @@
 # Order is important. The last matching pattern takes the most precedence.
 
 # Default owners for everything in the repo.
-*        @elixirschool/developers
+*        @elixirschool/english
 
-en/      @elixirschool/english
+lessons/en/      @elixirschool/english
 
-es/      @elixirschool/spanish
+lessons/es/      @elixirschool/spanish
 
-fr/      @elixirschool/french
+lessons/fr/      @elixirschool/french
 
-gr/      @elixirschool/greek
+lessons/gr/      @elixirschool/greek
```

```suggestion
lessons/el/      @elixirschool/greek
```

Since we just updated gr -> el